shotgun_api3/shotgun.py

```diff
@@ -29,36 +29,33 @@
  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 """
 
+# Python 2/3 compatibility
```

Comment:
also, please do a pass on the README file, to see that things in there still make sense.

Are the 'high performance requirements' still an option with the py3 changes?

Reply:
we should add a section to explain special gotchas when developing for py2/py3, like string/unicode wrappers via `six`.